elements/dat-import.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+const encoding = require('dat-encoding')
+const icon = require('./icon')
+const yo = require('choo/html')
+const css = require('sheetify')
+
+css('dat-colors')
+
+const prefix = css`
+  :host {
+    --icon-height: 1.2rem;
+    color: var(--color-neutral-30);
+  }
+
+  .icon-link {
+    padding-top: .32rem;
+    padding-left: .5rem;
+    pointer-events: none;
+    width: var(--icon-height);
+    height: var(--icon-height);
+  }
+  input {
```

Comment:
beware that this will style __all__ `<input>` fields. If you wanna make it private, prefix it with `:host > input` or something